### PR TITLE
ROS Docker and fix pointgrey build issue

### PR DESCRIPTION
**Pull request by RustyBamboo**
Dockerfile: 
- Install a base ubuntu system
- Creates a mil account
- wget install script, which is to be run by user once starting a container

Python Script (ros_docker_controller)
- Detects no images and automatically builds ROS Image
- ncurses interface to start a new container from ROS Image
- check if existing (even stopped) ROS container is avaliable and loads it
- ncurses interface to open a terminal instance or quit

### CODE REVIEW

File: ros_docker_controller/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# ROS Docker Controller
```
Comment (Grymessa) [inline ros_docker_controller/README.md:1]: Point them to the docker installation.

File: ros_docker_controller/docker_controller.py [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        # attempt to find what kind of terminal emulator is being used
+        self.terminal = None
+        self.find_avaliable_terminal()
+        assert self.terminal is not None, 'No terminal found'
+
+        # run curses stuff
+        self.draw()
+
+    # Finds terminal emulator
+    def find_avaliable_terminal(self):
```
Comment (Grymessa) [inline ros_docker_controller/docker_controller.py:60]: I'm fairly confident there is a better way to do this, like asking the user which terminal they prefer then remembering a default. Just launching a terminal seems... arbitrary.

**Review by Grymessa — CHANGES_REQUESTED** [review]
Excellent work, minor gripe about how you handle terminals though that I would like to see fixed before this is merged.

File: ros_docker_controller/README.md (hunk 1 above)
Comment (RustyBamboo) [reply ros_docker_controller/README.md:1]: Done

File: ros_docker_controller/docker_controller.py (hunk 2 above)
Comment (RustyBamboo) [reply ros_docker_controller/docker_controller.py:60]: Nah fam

**Review by Grymessa — APPROVED** [review]
I'll allow it. I'd prefer instead of asking the user every time what type of terminal you just store a default terminal text file somewhere that is loaded. If it isn't, ask user for preference. But this will work well as it stands.

**Pull request merged**